README.md

```diff
@@ -217,6 +217,7 @@ The supported rules are described below:
 | --------------------------- | ----------------------------------------------------------------------------- | -------- |
 | invalid_type_format_pair    | Flag any schema that does not follow the [data type/format rules.][2]         | shared   |
 | snake_case_only             | Flag any property with a `name` that is not lower snake case.                 | shared   |
+| no_schema_description       | Flag any schema without a `description` field.                                | shared   |
```

Comment:
I don't understand what is going on here.  This line is _already_ in `master`, so why is GHE showing this in the diff?